Dockerfile

```diff
@@ -7,9 +7,10 @@ RUN apt-get update && \
         ca-certificates \
         wget \
         gpg \
+        gpg-agent \
```

Comment:
1) If `gpg-agent` necessary for the gpg verification? 
2) If needed, then purge it on the `RUN` step on line 24. It should not be necessary for normal execution and the skinnier the image the better.

Reply:
1. Yes, without it the verification fails with the error that gpg-agent is unavailable.
2. You are right, I didn't pay attention to the remove part of the Dockerfile. Now that I look at it, seems like it can be optimized further. Every RUN command creates a separate layer, which means the files installed in the first layer stay within the image even when removed in the third layer. Putting everything in one layer (and one RUN) solves this issue and reduced the total size from 203MB down to 149MB.

Reply:
That's even better!